Integrations/integration-Wildfire.yml

```diff
@@ -254,10 +254,35 @@ script:
     description: Retrieve results for a file hash using WildFire
   - name: file
     description: Retrieve results for a file hash using WildFire
-    deprecated: true
```

Comment:
@slavikm I think the Behavior was weird for Wildfire as a !file, because if it is not a known hash, it takes it lot of time to respond (or respond "no results yet") unlike threat intelligence feeds.